packages/dashboard-frontend/src/pages/GetStarted/__tests__/GetStarted.spec.tsx

```diff
@@ -94,6 +91,7 @@ describe('Quick Add page', () => {
     await waitFor(() => expect(screen.getByRole('button', { name: 'Dummy Devfile' })).toBeTruthy());
 
     const devfileButton = screen.getByRole('button', { name: 'Dummy Devfile' });
+    await waitFor(() => expect(devfileButton).toBeTruthy());
```

Comment:
`getBy*` throws if no elements match.

```suggestion
    const devfileButton = await screen.findByRole('button', { name: 'Dummy Devfile' });
```

Reply:
I have fixed it.